migrations/20210216155935_initial.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable func-names */
+exports.up = function(knex) {
+  // Add extention for handling uuids to postgres
+  return knex.raw('CREATE EXTENSION IF NOT EXISTS "uuid-ossp";').then(function() {
+    return knex.schema
+      .createTable('universities', (table) => {
+        table.uuid('id').primary().defaultTo(knex.raw('uuid_generate_v4()'))
+        table.string('name', 255).notNullable()
+        table.timestamps() //updated_at / created_at
+      })
+  }).then(function() {
+    return knex.schema.createTable('patients', (table) => {
+      table.uuid('id').primary().defaultTo(knex.raw('uuid_generate_v4()'))
+      table.string('firstNames', 255).notNullable()
```

Comment:
```suggestion
      table.text('firstNames', 255).notNullable()
```
les premiers prenoms peuvent être longs